wg-notebooks/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Notebooks Working Group
+
+WG Notebooks is responsible for the user experience around Notebooks and other web IDEs and their integrations with Kubeflow
+
+The [charter](charter.md) defines the scope and governance of the Notebooks Working Group.
+
+## Meetings
+* Regular Notebooks Meeting (Asia & Europe friendly): [Thursdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Thursday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00 am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/10YCWL1l81B_uD1oWYGlbKPKhWugxcRnMUxcdCOZgHRs/edit).
+* Regular Notebooks Meeting (US friendly): [Thursdays at 4:00 pm UTC (Coordinated Universal Time)]() (monthly - fourth Thursday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=4:00 pm&tz=UTC%20%28Coordinated%20Universal%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/10YCWL1l81B_uD1oWYGlbKPKhWugxcRnMUxcdCOZgHRs/edit).
+
+## Organizers
+
+* Ilias Katsakioris (**[@elikatsis](https://github.com/elikatsis)**), Arrikto
```

Comment:
I'd love to see at least one non-Arrikto sponsor for this WG to improve diversity.

Reply:
@theadactyl, while I am not from a big vendor, I believe I would be a good "diversity pick"

I recently have been working lots on the notebooks with https://github.com/kubeflow/kubeflow/pull/5280 as an example.

Reply:
My primary expectation for leads is that they are committed as opposed to merely involved in a project ([ref](https://jakecalabrese.com/agile-commitment-scrum-pig-chicken-part-1/)).

My expectations is that the leads for a WG are committed to the long-term success of their projects. 

I agree with @theadactyl that all else being equal having multiple organizations is healthier and more resilient. Being committed (as opposed to merely involved) is a prerequisite for any lead.

w.r.t to bootstrapping the leads for leads; the things I'm looking for are

1. Demonstrated understanding and contribution to projects in question
1. Consensus among the proposed leads
1. Acknowledgement of the expectations for leads

Reply:
We are fine including @thesuperzapper in the WG to boost diversity, if he is committed to contribute to the project long term, per Jeremy's comments.

Reply:
Where did we land on this?